PractiseJune14/CarCollection.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace PractiseJune14
+{
+    public class CarCollection<T> : MyList<T> where T : ICar, new()
+    {
+        public T this[string indexName]
+        {
+            get
+            {
+                T result = default(T);
+                if (String.IsNullOrEmpty(indexName))
+                {
+                    throw new ArgumentException();
+                }
+                else
```

Comment:
Else is excessive since we throw exception and stop execution of this code in 'if' statement

Reply:
fixed